### ISSUE TITLE
UI behavior in Safari

### ISSUE THREAD

**Issue by iancorbitt** [mescon/Muximux#125]
OSX 10.12.4 Beta
Safari Version 10.1 (12603.1.23.1)

All the tabs are supposed to have text on them, but they don't. You can click on the tab image and the tab loads however.

<img width="1439" alt="screenshot 2017-02-12 12 45 58" src="https://cloud.githubusercontent.com/assets/1625407/22864541/6fbe98c0-f121-11e6-955e-6bbde7e829d4.png">

**Comment by mescon** [comment]
Very weird... can you check the source and see that the text is actually there?
I don't have OSX or Safari, so it's kinda hard to debug for me.

**Comment by int3nz0r** [comment]
I'm having the same issue, macOS 10.12.3 + Safari 10.0.3 (12602.4.8)

**Comment by iancorbitt** [comment]
Verified the text is actually in the source. I'm no web dev but it looks like the JS is causing the issue?

<img width="1341" alt="screenshot 2017-02-14 15 16 25" src="https://cloud.githubusercontent.com/assets/1625407/22947650/e463932a-f2c9-11e6-9211-9e4560178d6d.png">

**Comment by iancorbitt** [comment]
The full modal error message is:

Error while parsing the 'sandbox' attribute: 'allow-modals' is an invalid sandbox flag.

**Comment by iancorbitt** [comment]
Just for testing, I get identical errors in Safari Tech Preview 10.2

**Comment by mescon** [comment]
Seems that this error shouldn't cause the issue, according to this: https://github.com/jsfiddle/jsfiddle-issues/issues/882

**Comment by iancorbitt** [comment]
Makes sense. Not sure what the cause is then. I'm not getting any other errors in the Dev Console. I'll see what i can dig up.

**Comment by iancorbitt** [comment]
It's gotta be the CSS then...

Unfortunately I'm not versed in CSS enough to troubleshoot without help.

**Comment by shadowswan** [comment]
+1 for this, seems to work fine in chrome though

**Comment by pavstor** [comment]
This issue still remains and not only on macOS but also on iOS,

**Comment by mescon** [comment]
Muximux v2 (PHP) is no longer maintained. Muximux v3 is a complete rewrite — a standalone Go binary with a modern UI, built-in reverse proxy, themes, and 1,600+ icons. Please check the [main branch](https://github.com/mescon/Muximux) for the new version.
